eras/allegra/impl/testlib/Test/Cardano/Ledger/Allegra/ImpTest.hs

```diff
@@ -37,8 +40,22 @@ instance
 
   impSatisfyNativeScript = impAllegraSatisfyNativeScript
 
+class
+  ( ShelleyEraImp era
+  , NativeScript era ~ Timelock era
+  ) =>
+  AllegraEraImp era
+
+instance
+  ( Crypto c
+  , NFData (SigDSIGN (DSIGN c))
+  , NFData (VerKeyDSIGN (DSIGN c))
+  , Signable (DSIGN c) (Hash (HASH c) EraIndependentTxBody)
+  ) =>
+  AllegraEraImp (AllegraEra c)
```

Comment:
I added it initially for this: ` NativeScript era ~ Timelock era` and thought I'd add more things here,  but I'm removing it, since that was not the case, and it's not worth all the noise. I can add this constraint in the signature of the test function itself.